keyboards/ergodox_infinity/keymaps/nordic_ergo/keymap.c

```diff
@@ -175,82 +167,82 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-                KC_TRNS, KC_MPRV, KC_MSTP, KC_MPLY, KC_MNXT, KC_TRNS,
-       KC_TRNS, KC_WSCH, KC_WBAK, KC_WHOM, KC_WREF, KC_WFWD, KC_TRNS,
-                         KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
-       KC_TRNS, KC_TRNS,
-       KC_TRNS,
-       KC_TRNS, KC_WBAK, KC_WFWD
+   RESET,    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F11,
+   _______,  _______,  KC_BTN1,  KC_MS_U,  KC_BTN2,  _______,  _______,
+   _______,  KC_BTN4,  KC_MS_L,  KC_MS_D,  KC_MS_R,  KC_BTN5,
+   _______,  KC_WH_L,  KC_WH_D,  KC_WH_U,  KC_WH_R,  KC_BTN3,  _______,
+   _______,  _______,  KC_ACL0,  KC_ACL1,  KC_ACL2,
+                                           _______,  _______,
+                                           _______,
+                                           KC_BTN1,  KC_BTN2,  _______,
+   // right hand
+   KC_F12,   KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   _______,
+   _______, _______,   KC_VOLD,  KC_MUTE,  KC_VOLU,  _______,  _______,
+            _______,   KC_MPRV,  KC_MSTP,  KC_MPLY,  KC_MNXT,  _______,
+   _______, KC_WSCH,   KC_WBAK,  KC_WHOM,  KC_WREF,  KC_WFWD,  _______,
+   _______, _______,   _______,  _______,  _______,
+   _______, _______,
+   _______,
+   _______, KC_WBAK, KC_WFWD
 ),
 };
 
-const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
-{
-  // MACRODOWN only works in this function
-      switch(id) {
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
+    // MACRODOWN only works in this function
+    switch (id) {
         case 0:
-        if (record->event.pressed) {
-          SEND_STRING (QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
-        }
-        break;
+            if (record->event.pressed) {
+                SEND_STRING(QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
+            }
+            break;
         case 1:
-        if (record->event.pressed) { // For resetting EEPROM
-          eeconfig_init();
-        }
-        break;
-      }
+            if (record->event.pressed) {  // For resetting EEPROM
+                eeconfig_init();
+            }
+            break;
+    }
     return MACRO_NONE;
 };
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
-  switch (keycode) {
-    // dynamically generate these.
-    case EPRM:
-      if (record->event.pressed) {
-        eeconfig_init();
-      }
-      return false;
-      break;
-    case VRSN:
-      if (record->event.pressed) {
-        SEND_STRING (QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
-      }
-      return false;
-      break;
-    case RGB_SLD:
-      if (record->event.pressed) {
-        #ifdef RGBLIGHT_ENABLE
-          rgblight_mode(1);
-        #endif
-      }
-      return false;
-      break;
-  }
-  return true;
+    switch (keycode) {
+        // dynamically generate these.
+        case EPRM:
+            if (record->event.pressed) {
+                eeconfig_init();
+            }
+            return false;
+            break;
```

Comment:
This case isn't needed as the keycode isn't being used.

Also, QMK has a built-in EEPROM Reset keycode now:

https://github.com/qmk/qmk_firmware/blob/f033d8113d627c6950dc87936dfdb70e2b1ebf5f/quantum/quantum_keycodes.h#L458
https://github.com/qmk/qmk_firmware/blob/f033d8113d627c6950dc87936dfdb70e2b1ebf5f/quantum/quantum_keycodes.h#L626